app/commands/get_user_review_request_metrics.rb

```diff
@@ -0,0 +1,35 @@
+class GetUserReviewRequestMetrics < PowerTypes::Command.new(:github_user, :limit_month)
+  def perform
+    entry_count = 0
+    total_time = 0
+    review_requests_dic = Hash.new
+    limit_date = (Date.today - @limit_month.to_i.months).beginning_of_day
+    valid_pull_requests = @github_user.pull_requests
+                                      .where('gh_created_at > ?', limit_date)
+                                      .where.not(merged_by_id: nil)
+    valid_pull_requests.each do |pr|
+      if review_req = pr_has_review_requests(pr)
+        time_delta = (review_req.created_at - pr.gh_created_at).to_i
+        unless time_delta > 1.week
+          entry_count += 1
+          total_time += time_delta
+          review_requests_dic[pr.id] = { pr_title: pr.title, pr_created_at: pr.gh_created_at,
+                                        review_request_created_at: review_req.created_at,
+                                        time_delta: time_delta }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/HashAlignment: Align the keys of a hash literal if they span more than one line.